examples/booleansearchparser.py

```diff
@@ -151,7 +146,7 @@ def parser(self):
 
         #suport for non-western alphabets
         for r in alphabet_ranges:
-            alphabet += u''.join(_unichr(c) for c in range(*r) if not _unichr(c).isspace())
+            alphabet += u''.join(chr(c) for c in range(*r) if not chr(c).isspace())
```

Comment:
@ptmcg, @jdufresne   I agree with `chr`, but I believe it is still premature to remove all py2.  At least keep the imports at the top of the file...

```
    if PY2:
        chr = unichr
```

or maybe use `six` or some future (now past) compatibility lib?